pkg/sqlite/query.go

```diff
@@ -1117,3 +1117,71 @@ func (s *SQLQuerier) GetPropertiesForBundle(ctx context.Context, name, version,
 
 	return
 }
+
+func (s *SQLQuerier) GetTailForBundleInChannel(ctx context.Context, csvName string) (bundleNames []string, err error) {
+	getDepthQuery := `SELECT DISTINCT channel_name, package_name, depth FROM channel_entry WHERE operatorbundle_name=?`
```

Comment:
Should you just use `ORDER BY` or something like that to just sort this by `depth`?